gdal/ogr/ogr_srs_panorama.cpp

```diff
@@ -436,6 +438,18 @@ OGRErr OGRSpatialReference::importFromPanorama( long iProjSys, long iDatum,
             oGCS.importFromEPSG( aoDatums[iDatum] );
             CopyGeogCSFrom( &oGCS );
         }
+        else if( iEllips == PAN_ELLIPSOID_GSK2011 )
+        {
+            OGRSpatialReference oGCS;
+            oGCS.importFromEPSG( 7682 );
```

Comment:
7682 and 7680 are the codes for the Geographic 3D CRS, not the 2D ones. Is this intended ?

Reply:
Oo, yes! Thanks!